### ISSUE TITLE
Accessibility: Unread/read/forwarded/answered status not accessible with voice-over turned on

### ISSUE THREAD

**Issue by Falcopus1986** [signalapp/Signal-iOS#4750]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
When you are in a conversation and voice-over is turned on the Unread/read/forwarded/answered status of a message is not read by Voice-Over. 
### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
You get no information about the status of a message when Voice-over is turned on.
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
You should hear voice over say something like: Unread/read/forwarded/answered after reading the message. 
### Device info

**Device**: iPhone XS

**iOS version**: 14.3.0

**Signal version:** 5.2.5
